.github/workflows/haskell.yml

```diff
@@ -43,3 +43,5 @@ jobs:
       run: cabal build --enable-tests --enable-benchmarks all
     - name: Run tests
       run: cabal test all
+    - name: Build Docs
+      run: cabal haddock
```

Comment:
Travis was doing this, so this is a good idea if we want to be able to drop travis.